src/loaders/genome_collection/compute_genome_attribs.py

```diff
@@ -69,15 +69,32 @@
 
 from src.loaders.common import loader_common_names
 
-SERIES_TOOLS = []  # Tools cannot be executed in parallel
+SERIES_TOOLS = ['microtrait']  # Tools cannot be executed in parallel
 
 # Fraction amount of system cores can be utilized
 # (i.e. 0.5 - program will use 50% of total processors,
 #       0.1 - program will use 10% of total processors)
 SYSTEM_UTILIZATION = 0.5
 
+# source genome files can be missing for those collections
+# genomes with missing files will be skipped rather than raising an error
+# TODO parser script should catch those missing genomes and report them or writes to a file for later use
```

Comment:
is this something that'll happen soon are you thinking?

Reply:
It should be addressed with the rerun incomplete genomes ticket. Definitely not before the microtrait work. But we can bump the priority if you think it's more important.

Reply:
If we remove the empty directories this will be moot, so can be removed

Reply:
👍 updated todo.

Reply:
:+1: